src/blueapi/service/main.py

```diff
@@ -154,6 +157,24 @@ def submit_task(
         ) from e
 
 
+@app.get("/tasks")
+def get_tasks(
+    status: str = Query("unstarted", description="The status of the tasks to retrieve"),
```

Comment:
I think this should be a pydantic model with an enum rather than a string

Reply:
I forgor

Reply:
fastapi uses strings by default, I put an enum there. https://fastapi.tiangolo.com/tutorial/query-params/#query-parameter-type-conversion

Reply:
The good thing about a model is you can add to it without breaking anything else. It's also consistent with the rest of blueapi.

Reply:
https://github.com/DiamondLightSource/blueapi/pull/470#issuecomment-2126838045

do we want to force the users to type the request in ALL CAPS? 

because that's what we get if we expect the request to be an Enum